.gitpod.yml

```diff
@@ -1,3 +1,5 @@
+image: node:19.2.0
```

Comment:
Can you use the same node version as CircelCI and netlify in gitpod please ?

Reply:
Yes of course. I change it now

Reply:
done ✅